source/agora/common/Config.d

```diff
@@ -168,6 +168,12 @@ public struct NodeConfig
     /// is going to connect to, for example: http://0.0.0.0:8008
     /// It can also be set to 0 do disable listening
     public ushort stats_listening_port;
+
+    /// The maximum size of data payload
+    public uint max_size_payload;
+
+    /// The factor to calculate for the fee of data payload
+    public uint payload_fee_factor;
```

Comment:
Can we rename this to `tx_payload_fee_factor`?

Reply:
Also: It should have a default value, no?

Reply:
This is a good suggestion. I'll fix it.